spec/index.bs

```diff
@@ -168,7 +323,33 @@ interface GPUDevice : EventTarget {
 GPUDevice includes GPUObjectBase;
 </script>
 
-### Creation ### {#device-creation}
+{{GPUDevice}} also has the following internal slots:
+
+<dl dfn-type=attribute dfn-for=GPUDevice>
+    : <dfn>\[[device]]</dfn>, of type [=device=], readonly
+    ::
+        The [=device=] that this {{GPUDevice}} refers to.
+</dl>
```

Comment:
I presume the "[=device=] that this {{GPUDevice}} refers to" is meant to be an "internal slot"?

Reply:
Not sure I understand, but yes it is an internal slot:

> {{GPUDevice}} also has the following internal slots: